src/client/components/CommentFooter/Buttons.js

```diff
@@ -174,28 +176,30 @@ class Buttons extends React.Component {
             {pendingLike ? <Icon type="loading" /> : <i className="iconfont icon-praise_fill" />}
           </a>
         </Tooltip>
-        <span
-          className={classNames('CommentFooter__count', {
-            'CommentFooter__count--clickable': upVotes.length > 0 || downVotes.length > 0,
-          })}
-          role="presentation"
-          onClick={this.handleShowReactions}
-        >
-          <Tooltip
-            title={
-              <div>
-                {upVotesPreview}
-                {upVotesMore}
-                {upVotesPreview.length === 0 && (
-                  <FormattedMessage id="no_likes" defaultMessage="No likes yet" />
-                )}
-              </div>
-            }
+        {upVotes.length > 0 && (
+          <span
+            className={classNames('CommentFooter__count', {
+              'CommentFooter__count--clickable': upVotes.length > 0 || downVotes.length > 0,
+            })}
+            role="presentation"
+            onClick={this.handleShowReactions}
           >
-            <FormattedNumber value={upVotes.length} />
-            <span />
-          </Tooltip>
-        </span>
+            <Tooltip
+              title={
+                <div>
+                  {upVotesPreview}
+                  {upVotesMore}
+                  {upVotesPreview.length === 0 && (
```

Comment:
I think that this goes like with the conditions before, after checking upVotes.length > 0 it is not required to check this - it will be always false I suppose.

Reply:
Good points on both of these. Would you like me to go back and fix it?

Reply:
Sure, you can give it a go.